### DISCUSSION TITLE
a few suggestions

### DISCUSSION THREAD

**Discussion by umut2054** [orioncactus/pretendard#140] [Ideas]

**Comment by umut2054** [comment]
![20230617_150842](https://github.com/orioncactus/pretendard/assets/121461492/6567ee6c-2a59-43cd-8810-f7e41bbe1646)
![20230617_150905](https://github.com/orioncactus/pretendard/assets/121461492/3266c42f-e792-4e33-84c7-96280025cc43)
![20230618_010854](https://github.com/orioncactus/pretendard/assets/121461492/c18454bb-8df8-42c3-94f2-21764fcb6f73)
![20230617_150642](https://github.com/orioncactus/pretendard/assets/121461492/08b0cab1-c99f-4be7-a677-4a9fcda3e453)
![20230617_232235](https://github.com/orioncactus/pretendard/assets/121461492/997bd386-6e40-410d-8cf9-5a9f8c094c64)

**Comment by orioncactus** [comment]
Hello! Thank you for your valuable feedback regarding Pretendard.

As Pretendard is based on Inter as its foundation, there may be some differences compared to SF Pro Text. I will carefully consider implementing your suggestions to enhance readability. Additionally, I plan to adjust the spacing by approximately 0.1381% to match the body level (17pt on iOS) of San Francisco.

However, there are some higher priority tasks that need to be addressed first, so I will incorporate this feedback once those tasks are completed.
